src/sentry/api/event_search.py

```diff
@@ -122,14 +59,14 @@ class SearchKey(namedtuple('SearchKey', 'name')):
 
     @property
     def snuba_name(self):
-        field = FIELD_LOOKUP.get(self.name)
-        if field:
-            return field['snuba_name']
+        snuba_name = SEARCH_MAP.get(self.name)
+        if snuba_name:
+            return snuba_name
         # assume custom tag if not listed
         return 'tags[%s]' % (self.name,)
 
 
-class SearchValue(namedtuple('SearchValue', 'raw_value type')):
+class SearchValue(namedtuple('SearchValue', 'raw_value')):
```

Comment:
i wasn't actually using type for anything (thought it might have been helpful at one point) so just removed